cmd/image-builder/config_test.go

```diff
@@ -31,7 +31,7 @@ func TestEnv(t *testing.T) {
 
 	config := ImageBuilderConfig{
 		ListenAddress: "localhost",
-		LogLevel: "DEBUG",
+		LogLevel:      "DEBUG",
```

Comment:
this formatting and the similar ones are due to not running `tools/prepare-source.sh` in #46